src/test/java/org/sonar/plugins/clojure/sensors/ancient/AncientRulesTest.java

```diff
@@ -0,0 +1,28 @@
+package org.sonar.plugins.clojure.sensors.ancient;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.sonar.api.server.profile.BuiltInQualityProfilesDefinition;
+import org.sonar.plugins.clojure.language.ClojureSonarWayProfile;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static java.util.Arrays.asList;
+import static org.junit.Assert.assertTrue;
+
+public class AncientRulesTest {
```

Comment:
Is this class used?

Reply:
Not anymore it was left over after moving spesific rules test to one class. This is now removed